cmd/test_util.go

```diff
@@ -34,6 +34,21 @@ func FixTestSetup(t *testing.T, file string, auditFunction func(Resource) []Resu
 	return assert, fixPotentialSecurityIssue(resource, result)
 }
 
+// FixTestSetupMultipleResources allows kubeaudit to be used programmatically instead of via the shell for multiple Resources. It is intended to be used for testing.
+func FixTestSetupMultipleResources(t *testing.T, file string, auditFunction func(Resource) []Result) (*assert.Assertions, []Resource) {
```

Comment:
@klautcomputing  How do you feel about using this test everywhere we were using `FixTestSetup`?

Reply:
Yeah sounds good to me. I would go for the shorter name though :)